cpp/inc/bond/ext/grpc/server.h

```diff
@@ -118,12 +150,63 @@ namespace bond { namespace ext { namespace gRPC
             BOOST_ASSERT(_server);
             BOOST_ASSERT(_ioManager);
 
+            start();
+        }
+
+        void start()
```

Comment:
Nothing else calls start that I can see: keep this look inside the ctor?

Reply:
This way looks cleaner.